plugins/nf-lamin/src/main/nextflow/lamin/LaminObserver.groovy

```diff
@@ -138,118 +123,203 @@ class LaminObserver implements TraceObserver {
 
     // --- private methods ---
     protected void testConnection() {
-        if (!this.instance) {
-            throw new IllegalStateException('API client is null')
-        }
-
         String instanceString = "${this.instance.getOwner()}/${this.instance.getName()}"
         try {
-            this.instance.getInstanceStatistics()
-            log.info "Connected to Lamin instance: ${instanceString}"
+            this.instance.getNonEmptyTables()
+            log.info "✅ Connected to Lamin instance '${instanceString}'"
```

Comment:
Can we please call it LaminDB instance instead of Lamin instance?

Reply:
Fixed!